lms/djangoapps/ccx/views.py

```diff
@@ -552,7 +552,8 @@ def ccx_grades_csv(request, course, ccx=None):
                 }
 
                 row_percents = [percents.get(label, 0.0) for label in header]
-                rows.append([student.id, student.email, student.username,
+                rows.append([student.id, student.email.encode('utf-8'),
```

Comment:
I tried registering a user with non-ascii characters in the email, and the registration form wouldn't accept it.

That's no reason not to include this encoding, just FYI.